### PR TITLE
Parser tests

### PR DESCRIPTION
**Pull request by dentarg**
Test the parser in the same fashion as https://github.com/twingly/twingly-search-api-ruby/pull/49

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
     "url": "git://github.com/twingly/twingly-search-api-node.git"
   },
   "dependencies": {
+    "chai-datetime": "^1.4.0",
```
Comment (walro) [inline package.json]: shouldn't these be dev dependencies?

Comment (dentarg) [inline package.json]: Indeed they should, I used `--save` when I should have used `--save-dev`, will fix, thanks

**Comment by walro** [comment]
LGTM

File: test/test_result.js [hunk 2]
```diff
@@ -2,11 +2,13 @@ var expect = require('chai').expect;
 var fs = require('fs');
 
 var Parser = require('../lib/parser');
+var Result = require('../lib/result');
 
 describe('result', function(){
     it('creation', function(done){
         var data = fs.readFileSync('./test/fixtures/valid_result.xml', {encoding: 'utf8'});
         (new Parser()).parse(data, function(error, result){
+            expect(result).to.be.instanceof(Result);
```
Comment (walro) [inline test/test_result.js]: Don't think this is needed here, the point is not to check the return value of the parser but to assert that we find certain properties on the Result type (this is a test for `Result`).

Comment (dentarg) [inline test/test_result.js]: Yes, that make sense, thanks.

**Comment by walro** [comment]
LGTM again :D

**Comment by dentarg** [comment]
:smile:

**Pull request merged**